docker/docker-compose.yml

```diff
@@ -104,3 +121,16 @@ volumes:
   # during ltcd container recreation.
   litecoin:
     driver: local
+
+  # lnd volumes are needed for maintaining lightning channels persistance
+  # during lnd_* containers recreation.
+  lnd_ltc_data:
+     driver: local
+  lnd_btc_data:
+     driver: local
+
+  # alice and bob are need for running README examples
+  alice_data:
```

Comment:
IMO this `docker-compose` file should be left agnostic to implementation examples.